lib/jsdom/living/xmlhttprequest.js

```diff
@@ -1090,9 +1090,10 @@ module.exports = function createXMLHttpRequest(window) {
 
   function setResponseToNetworkError(xhr) {
     const properties = xhr[xhrSymbols.properties];
+    properties.responseCache = properties.responseTextCache = properties.responseXMLCache = null;
+    properties.responseHeaders = {};
     properties.status = 0;
     properties.statusText = "";
-    properties.responseCache = properties.responseTextCache = properties.responseXMLCache = null;
```

Comment:
Note: This line is just me taking the opportunity to sort the properties in alphabetical order. Let me know in case you'd like me to avoid this in the future.